.github/stale.yml

```diff
@@ -1,7 +1,7 @@
 # Number of days of inactivity before an issue becomes stale
-daysUntilStale: 210
+daysUntilStale: 30
 # Number of days of inactivity before a stale issue is closed
-daysUntilClose: 15
+daysUntilClose: 60
 # Issues with these labels will never be considered stale
 exemptLabels:
   - pinned
```

Comment:
I was thinking of adding few more labels here for which stale rules wont apply... for example: `assigned`. `enhancement` etc... (if it makes sense).

Reply:
I agree we should use some label to prevent issues from becoming stale and being closed. But I'm not sure if `enhancement` is a good option (I would say that bugs should have higher priority than enhancements, so they should also be kept open).

I think the key point is to know if someone is working on them. If so, I think it's fine to keep them open. Otherwise we should close them, whatever it is. There could be some exceptions though that also require to keep the issue opened, so maybe a new specific label for this purpose would be cleaner ?

Reply:
For now, added a label [`assigned`](https://github.com/gluster/glusterfs/labels/assigned) which is exempted from stable bot.